quantum/nodes/cnot-gate/cnot-gate.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+  function CNotGateNode(config) {
+    RED.nodes.createNode(this, config);
+    this.name = config.name;
+    this.qubits = [];
+    this.targetPosition = config.targetPosition;
+    const node = this;
+
+    node.on('input', async function(msg, send, done, targetQubit, controlQubit) {
```

Comment:
How is the input listener receiving the data for `targetQubit` and `controlQubit`? I thought the signature for the callback function here had to be in the format `this.on('input', function(msg, send, done) { ... }` (as described [here](https://nodered.org/docs/creating-nodes/node-js#receiving-messages)), or am I missing something?

Reply:
Just looking at the online documentation, the input event callback function should follow the format function(msg, send, done) { ... }, thank you for your comment, will fix it